src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -98,6 +98,29 @@ export class QuestionSubmissionFormComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.sortRecipientsByName();
```

Comment:
Can I check whether you've tried running this and checking that it works in the dev environment? I'm asking because it seems you're sorting on the component initialisation (`ngOnInit()`) whereas the way this component has been written seems to suggest that all model manipulation should be handled in the model `@Input`. The way your tests have been written manually set the model before calling `ngOnInit()` which might not be the call order when the component is mounted. I'm not very sure on this so wanted to check if it has been manually tested.

Reply:
I've added a side by side video comparison and added it to the PR comment

Reply:
Perfect, thanks!